Prevent a memory leak in call-channel, which otherwise would cause a
test failure in file-transfer-channel.c.

Taken from upstream:
https://gitlab.freedesktop.org/telepathy/telepathy-glib/commit/2e8b2cc0db09b781d99e3755bb6d01548037fe7b

diff --git a/telepathy-glib/call-channel.c b/telepathy-glib/call-channel.c
index 8ed76323be32f83c5f4d6045a50bc1698d8093fd..2a1fa203d4be9f0c8b99344225f4ba9900666ffc 100644
--- a/telepathy-glib/call-channel.c
+++ b/telepathy-glib/call-channel.c
@@ -525,6 +525,8 @@ update_call_members (TpCallChannel *self,
 
   _tp_channel_contacts_queue_prepare_async ((TpChannel *) self,
       contacts, update_call_members_prepared_cb, data);
+
+  g_ptr_array_unref (contacts);
 }
 
 static void
